framework/helpers/BaseConsole.php

```diff
@@ -59,7 +59,7 @@ class BaseConsole
      */
     public static function moveCursorUp($rows = 1)
     {
-        echo "\033[" . (int) $rows . 'A';
+        echo "\033[" . (int)$rows . 'A';
```

Comment:
the other changes are fine but this one is part of any standard or code style used by yii2?